build_images.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+foreman_version=$1
+
+podman build images/centos8 -t smoker-test/centos8:$foreman_version --build-arg foreman_version=$foreman_version
```

Comment:
Food for thought: should we use labels to indicate capabilities? Like a katello-agent label. That way in the future we can add Debian clients where there is no katello-agent, but we can test content. Not something that needs to be implemented now